src/main/java/com/carrotcreative/recyclercore/inject/InjectController.java

```diff
@@ -0,0 +1,18 @@
+package com.carrotcreative.recyclercore.inject;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Created by kaushiksaurabh on 10/1/15.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface InjectController
```

Comment:
Could we use two different `@interface`s to have one for view and then controller?

Don't know if this is possible:

```java
@Controller(UserListRecyclerController.class)
@Layout(R.layout.element_user_list)
public class MyModel extends RecyclerCoreModel
{
     // ...
}
```

Although I'm likely to think that this can be done because something similar is done in Retrofit.

Although you may want it to be one interface.  Just throwing that idea out there